### ISSUE TITLE
[clang-format] Space before function name if cast function return value

### ISSUE THREAD

**Issue by bo-wu** [llvm/llvm-project#86638]
Hello everyone, 

I set  **SpaceAfterCStyleCast: false** in my config file and it works for C style data type cast.
However, when I directly cast function return value, there always exists a space like  (uint32) Func(), but not (uint32)Func().

which config item is for that? 
Thanks in advance.

**Comment by owenca** [comment]
Try adding `uint32` to `TypeNames`. Using the trunk version (5bb3492892f0e087249d6e7aa9c077b9aa0922eb):
```
$ echo '(uint32) Func();' | clang-format -style='{TypeNames: [uint32]}'
(uint32)Func();
```

**Comment by owenca** [comment]
This issue is addressed by 0baef3b18cdbb.

**Comment by bo-wu** [comment]
> Try adding `uint32` to `TypeNames`. Using the trunk version ([5bb3492](https://github.com/llvm/llvm-project/commit/5bb3492892f0e087249d6e7aa9c077b9aa0922eb)):
> 
> ```
> $ echo '(uint32) Func();' | clang-format -style='{TypeNames: [uint32]}'
> (uint32)Func();
> ```

Thank you very much!
I use the clang-format config file. And add the type under TypeNames item also solve the problem.
